pycytominer/normalize.py

```diff
@@ -162,14 +162,26 @@ def normalize(
         fitted_scaler = scaler.fit(profiles.query(samples).loc[:, features])
 
     # Scale the feature dataframe
+    # See https://github.com/cytomining/pycytominer/issues/319
+    fitted_scaled = fitted_scaler.transform(feature_df)
+    if type(fitted_scaled) == pd.DataFrame:
+        fitted_scaled = fitted_scaled.values
     feature_df = pd.DataFrame(
-        fitted_scaler.transform(feature_df),
+        fitted_scaled,
         columns=feature_df.columns,
         index=feature_df.index,
     )
 
     normalized = meta_df.merge(feature_df, left_index=True, right_index=True)
 
+    assert (
+        feature_df.shape == profiles.loc[:, features].shape
+    ), "Feature dataframe shape does not match original dataframe shape"
+
+    assert (normalized.shape[0] == profiles.shape[0]) and (
+        normalized.shape[1] == len(features) + len(meta_features)
+    ), "Normalized dataframe shape does not match original dataframe shape"
```

Comment:
I like these checks, but they could be frustrating errors for our users. Is there a way to be more specific about what to check if this assertion fails?

Reply:
The purpose of this assertion is really to identify new test fixtures, because this indicates a bug in software. I now prompt the user to file an issue.

Reply:
I now use `create_bug_report_message`